### PR TITLE
linter: added classMembersOrder checker

### PR DESCRIPTION
**Pull request by i582**
`classMembersOrder` checks that all constants and
properties of the class go before methods.

### CODE REVIEW

File: src/linter/report.go [hunk 1]
```diff
@@ -671,6 +671,23 @@ function f(array $a) {}`,
 			Before:   `strpos('/', $s);`,
 			After:    `strpos($s, '/');`,
 		},
+
+		{
+			Name:     "classCompOrder",
+			Default:  false,
```
Comment (i582) [inline src/linter/report.go:677]: Not enabled by default.

**Comment by quasilyte** [comment]
Conflicts.

**Comment by i582** [comment]
> Conflicts.

Fixed.

**Review by quasilyte — APPROVED** [review]

File: src/tests/checkers/oop_test.go [hunk 2]
```diff
@@ -1529,3 +1529,98 @@ function a1(Foo $a, foo $b, boo $c) {}
 	}
 	test.RunAndMatch()
 }
+
+func TestClassComponentsOrder(t *testing.T) {
```
Comment (quasilyte) [inline src/tests/checkers/oop_test.go:1533]: Wasn't it intended to be a "Members" order?

**Pull request merged**